test/passes/remove-non-js-ops.txt

```diff
@@ -33,39 +33,15 @@
  (global $__wasm-intrinsics-temp-i64 (mut i64) (i64.const 0))
  (memory $0 1)
  (func $copysign64 (param $0 f64) (param $1 f64) (result f64)
-  (f64.reinterpret_i64
-   (i64.or
-    (i64.and
-     (i64.reinterpret_f64
-      (local.get $0)
-     )
-     (i64.const 9223372036854775807)
-    )
-    (i64.and
-     (i64.reinterpret_f64
-      (local.get $1)
-     )
-     (i64.const -9223372036854775808)
-    )
-   )
+  (f64.copysign
+   (local.get $0)
+   (local.get $1)
```

Comment:
These test functions can probably be removed entirely now that they don't test anything interesting.

Reply:
yeah, I'll move this fn to new test